vm_scripts/prepare_vapt_lab.py

```
@@ -0,0 +1,221 @@
+                extra = {
+                    'machine': "NA",
+                    'rdp_ip': "NA",
+                    'username': "NA",
+                    'password': "NA",
+                    'browser_url': ""
+                }
+
+                username = User.objects.get(username=ele['user'])
+                sender = User.objects.get(username=kwargs["username"])
+                notify.send(sender, recipient=username, actor=sender,
+                            verb="VAPT Server Assigned ".format(kwargs["username"]),
```

Ruff (0.8.2)

`.format` call has unused arguments at position(s): 0

Remove extra positional arguments at position(s): 0

(F523)